azclishell/app.py

```diff
@@ -393,7 +393,7 @@ def _special_cases(self, text, cmd, outside):
         if self.default_command:
             cmd = self.default_command + " " + cmd
 
-        if text.strip() == "quit" or text.strip() == "exit":
+        if cmd.strip() == "quit" or cmd.strip() == "exit":
```

Comment:
I use cmd to mean more of what the user wanted as a command versus the text is what's on the line. In this specific example, I strip 'az' from the cmd but not the text, so what I to know is if they wrote 'az quit' versus if the line said 'quit.'